src/LaraCart.php

```diff
@@ -450,7 +451,7 @@ public function taxTotal($format = true)
 
         foreach ($this->getFees() as $fee) {
             if ($fee->taxable) {
-                $totalTax += $fee->amount * $this->cart->tax;
+                $totalTax += $fee->amount * $fee->tax;
```

Comment:
A test doesn't pass without it. And this allow to set different tax rate separately.